docs/azure_jumpstart_ag/contoso_supermarket/ai/_index.md

```diff
@@ -6,6 +6,52 @@ Contoso prides itself on being technology innovators. In the current market, Con
 
 To this end, Contoso has developed an AI mechanism that empowers them to enhance the customer experience in their stores. The objective is to accurately detect the number of people waiting at different checkout lanes and enable proactive actions based on this data.
 
+## Architecture
```

Comment:
I am missing some context on the AI portion. What technology is being used and why.